bloom/bloomfilter.go

```diff
@@ -44,22 +47,7 @@ func NewBloomFilterLegacy(m, h uint) (BloomFilter, error) {
 	}
 }
 
-// BloomFilterFromBytesLegacy constructs a bloom filter from bytes
-func BloomFilterFromBytesLegacy(b []byte, m, h uint) (BloomFilter, error) {
-	switch m {
-	case 2048:
-		return bloom2048FromBytes(b, h)
-	default:
-		return nil, errors.Errorf("bloom filter size %d not supported", m)
-	}
-}
-
 // NewBloomFilter returns a new bloom filter
 func NewBloomFilter(m, h uint64) (BloomFilter, error) {
 	return newBloomMbits(m, h)
 }
-
-// BloomFilterFromBytes constructs a bloom filter from bytes
-func BloomFilterFromBytes(buf []byte) (BloomFilter, error) {
```

Comment:
this is also needed in the bloom filter indexer

Reply:
see https://github.com/iotexproject/iotex-core/pull/3074